api-spec/definitions/WFS3core.fragment.yaml

```diff
@@ -0,0 +1,385 @@
+paths:
```

Comment:
I did NOT review this file as I hope that it is just taken from the WFS spec and doesn't need further review.

Reply:
I assume we'll update this anyway every now and then to be compatible with WFS.